cmd/up/up.go

```diff
@@ -425,7 +431,7 @@ func (up *upContext) buildDevImage(ctx context.Context, app apps.App) error {
 	if _, err := os.Stat(up.Dev.Image.Dockerfile); err != nil {
 		return oktetoErrors.UserError{
 			E:    fmt.Errorf("'--build' argument given but there is no Dockerfile"),
-			Hint: "Try creating a Dockerfile or specify 'context' and 'dockerfile' fields.",
+			Hint: "Try creating a Dockerfile field or specify the 'context' and 'dockerfile' fields in your okteto manifest.",
```

Comment:
```
"Try creating a 'dockerfile' field or specify...
```

Is the field name `dockerfile`?

Reply:
updated. it's a typo, it should be Dockerfile file